### DISCUSSION TITLE
Deployment problem

### DISCUSSION THREAD

**Discussion by ChengJiuSu** [civitai/civitai#311] [General]
For the.env-example configuration item, I want to know what parameter `S3_ORIGINS` stands for? It would be appreciated if it could be resolved

**Comment by WASasquatch** [comment]
AWS CloudFront origins, pretty sure.

https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/DownloadDistS3AndCustomOrigins.html

**Reply by ChengJiuSu** [reply]
thank you very much for your help

**Comment by ChengJiuSu** [comment]
There is a new problem now, and I start service after configuration. When uploading the picture, it will be stuck. Finally, it is confirmed that there is no triggering call/API/Image-UPLOAD interface. It has been verified that the configuration and interface status has always been normal, so I want to continue to ask what it may affect
